core/templates/pages/topic-viewer-page/topic-viewer-page.controller.ts

```diff
@@ -57,6 +57,7 @@ angular.module('oppia').directive('topicViewerPage', [
           ctrl.checkMobileView = function() {
             return (WindowDimensionsService.getWidth() < 500);
           };
+          ctrl.canonicalStoriesList = [];
```

Comment:
Any specific reason to define this here? (not inside the onInit method)

Reply:
No reason as such.